full_requirements.txt

```diff
@@ -0,0 +1,4 @@
+allennlp
```

Comment:
what's the idea here? Will full_req.txt and req.txt live together? What's their difference and how does one invoke one vs. the other on pip install

Reply:
These are our optional requirements. When you do `pip install textwiser`, you would install our regular requirements.txt. If you do `pip install textwiser[full]`, it would install both requirements.txt and full_requirements.txt. If you do the full installation you won't have to go and individually install tensorflow for USE, UMAP, etc.

Reply:
If you don't use `pip install textwiser[full]`, you cannot use ELMo, USE, UMAP, etc. All of these are in our optional dependencies, in an attempt to make the library "lighter". The library does work without them, it will simply give you an error if you try to initialize a USE model without having tensorflow. Tests also pass without them, those tests are not run if you don't have the corresponding library.

This is in install page, I'll add to readme as well.

Reply:
yeah a one-liner in the Readme to expose the availability of [full] and what it does should do it